pkg/sqlite/load.go

```diff
@@ -645,6 +672,8 @@ func (s *sqlLoader) getCSVNames(tx *sql.Tx, packageName string) ([]string, error
 	return csvNames, nil
 }
 
+// RemovePackage is a method responsible for removing a
```

Comment:
`nit`: Might be worth mentioning that the package is removed in it's entirety, i.e channels/CSVs etc removed from other tables too.

Reply:
Yeah, makes sense - done in the last round of force pushes.